contrib/oauth2_proxy.cfg.example

```diff
@@ -10,7 +10,7 @@
 # tls_key_file = ""
 
 ## the OAuth Redirect URL.
-# defaults to the "https://" + requested host header + "/oauth2/callback"
+# defaults to "https://" + requested host header + "/oauth2/callback"
```

Comment:
Nit: Other comments in this example use `##` to start the line, separating the comments from the example config lines that are commented out, we should stay consistent
```suggestion
## defaults to "https://" + requested host header + "/oauth2/callback"
```